src/deadwood/cog.py

```diff
@@ -54,13 +55,26 @@ def _gdal_calculate_cog(tiff_file_path: str, cog_target_path: str, compress="jpe
         "-co", 'TILING_SCHEME=GoogleMapsCompatible',
         "-co", "OVERVIEWS=IGNORE_EXISTING",
         "--config", "GDAL_TIFF_INTERNAL_MASK", "TRUE",
-        
+        "--config", "GDAL_GTIFF_SRS_SOURCE", "EPSG"
     ]
     if overviews is not None:
         cmd_translate.extend(["-co", f"OVERVIEW_COUNT={overviews}"])
 
     # apply
-    subprocess.run(cmd_translate)
+    try:
+        logger.info(f'Running gdal_translate with EPSG:3857', extra={'token': token})
```

Comment:
This seems wrong? You are not translating with EPSG 3857 here, no?

Reply:
Yes, in the first attempt i try to run let gdal_translate handle the conversion from x to EPSG:3857.
But in case of an error, I assume its EPSG:3857 and set is explicitly. 
The translation target is set by `TILING_SCHEME=GoogleMapsCompatible`  (its  EPSG:3857)
the problem is that in some cases the source crs is not correctly identified.

Reply:
Why from x to EPSG:3857. Don't you want 4326 for the COG?

Reply:
No, all web mapping libraries use 3857. In the background, `TILING_SCHEME=GoogleMapsCompatible` optimizes cogs for visualization hence, converts to 3857 and creates overviews best for the tiling system of the mapping providers.

Reply:
The logging behind the approach above is, to handle the problem that there will be cases where the source crs is unkown or corrupted. In this case, we first try the default way of things, next we try out different source crs based on the already experiences bugs and errors we had. So in the future if a new bug with another crs is found we could simply add it as one additional try.

Of course, we need to carefully check that the results are correct.

Reply:
okay. sounds good. Don#t have the time today to look into this. But generally sounds good